.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk7
+script:
+  - mvn clean install verify -Dgpg.skip=true
```

Comment:
AFAIK you should use

    install: mvn install -DskipTests -Dgpg.skip

and remove the `script` section